scripts/quick-start-koku.sh

```
@@ -1012,6 +1009,50 @@ PY
+  local current
+  current=$(countKokuWorkers)
+
+  if [ "$current" -ge "$desired" ]; then
+    echo "    already have ${current} worker(s) (want ${desired}); skipping scale"
+    return 0
+  fi
+
+  echo "    scaling koku-worker ${current} → ${desired} (timeout ${timeout_s}s)..."
+  # perl alarm: portable timeout without requiring GNU coreutils `timeout`.
+  if ! perl -e 'alarm shift; exec @ARGV' "$timeout_s" \
+    $PODMAN_COMPOSE -f docker-compose.yml -f "$PODMAN_OVERRIDE_FILE" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)